src/validate.ts

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+chai.use(require('chai-json-schema-ajv'));
+
+function generateRandom(min, max){
+    //helper function for obtaining a random requestId
+    return Math.floor(Math.random() * (max-min) + min).toString();
+}
+
+
+function generateSyncReq(){
+    const requestId = generateRandom(100,999);
+    return {
+        requestId,
+        "inputs": [{
+          "intent": action.devices.SYNC
+            }]
+    }
+}
+
+const sync_request = generateSyncReq();
+const requestRandId = sync_request["requestId"];
+
+function generateQueryReq(deviceID){
```

Comment:
Is `deviceID` an array? If so, we might want to be more explicit with the naming (e.g. deviceIds, targetDevices)